content/docs/deploy/core/upgrading.mdx

```diff
@@ -13,6 +13,26 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## 0.25.0
+
+### Breaking
+
+#### Base64-encoded Certificates
+
+Previously, the `certificates` key supported base64-encoded certificates as a value (this option was not documented). We've removed support for base64-encoded certificates for this setting and now require that you only use the certificate file location. To avoid incompatabilities between versions, please update these values accordingly.
```

Comment:
I think cspell is correct here; this should be "incompatibilities".

```suggestion
Previously, the `certificates` key supported base64-encoded certificates as a value (this option was not documented). We've removed support for base64-encoded certificates for this setting and now require that you only use the certificate file location. To avoid incompatibilities between versions, please update these values accordingly.
```